src/SourceBuild/content/repo-projects/source-build-reference-packages.proj

```diff
@@ -8,16 +8,24 @@
     <!-- SBRP builds before Arcade so it also needs the bootstrap Arcade version -->
     <UseBootstrapArcade>true</UseBootstrapArcade>
 
+    <!-- SBRP builds the reference SDKs so it needs the bootstrap SDK versions -->
+    <UseBootstrapSdks>true</UseBootstrapSdks>
+    
+    <ExtractToolPackageDependsOn>_DiscoverToolPackages</ExtractToolPackageDependsOn>
+
     <LocalNuGetPackageCacheDirectory>$(ArtifactsObjDir)source-build-reference-package-cache</LocalNuGetPackageCacheDirectory>
 
     <BuildArgs>$(BuildArgs) /p:MicrosoftNetCoreIlasmPackageRuntimeId=$(NETCoreSdkRuntimeIdentifier)</BuildArgs>
     <BuildArgs>$(BuildArgs) /p:LocalNuGetPackageCacheDirectory=$(LocalNuGetPackageCacheDirectory)</BuildArgs>
   </PropertyGroup>
-
-  <ItemGroup>
-    <BuiltSdkPackageOverride Include="@(NoTargetsSdkOverride)" />
-    <BuiltSdkPackageOverride Include="@(TraversalSdkOverride)" />
-  </ItemGroup>
+  
+  <Target Name="_DiscoverToolPackages" 
+          DependsOnTargets="RepoBuild;GetProducedPackages">
```

Comment:
From looking at this target, it only depends on `GetProducedPackages` which I then would expect to depend on `RepoBuild`.